src/DumpRecorder/DumpRecorder.php

```diff
@@ -18,9 +20,15 @@ class DumpRecorder
 
     protected static $registeredHandler = false;
 
+    protected static $runningLaravel9 = null;
+
     public function __construct(Container $app)
     {
         $this->app = $app;
+
+        if (static::$runningLaravel9 === null) {
```

Comment:
You can check the current Laravel version with 

```
app()->version();
```

Use `version_compare` if it's equal or higher than 9.0.0: https://www.php.net/manual/en/function.version-compare.php

Reply:
Much better solution, thank you!